lmfit/lineshapes.py

```diff
@@ -302,6 +302,41 @@ def expsine(x, amplitude=1.0, frequency=1.0, shift=0.0, decay=0.0):
     return amplitude*sin(x*frequency + shift) * exp(-x*decay)
 
 
+def thermal_distribution(x, amplitude=1.0, center=0.0, kt=1.0, form='Bose'):
```

Comment:
please use ```bose``` (i.e., not capitalized) here in the signature, since you use the same in the docstring(s)